sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnInvokers.java

```diff
@@ -153,25 +157,13 @@ public String subclass(TypeDescription.Generic superClass) {
             .method(ElementMatchers.named("invokeProcessElement"))
             .intercept(new ProcessElementDelegation(signature.processElement()))
             .method(ElementMatchers.named("invokeStartBundle"))
-            .intercept(
-                signature.startBundle() == null
-                    ? new NoopMethodImplementation()
-                    : new BundleMethodDelegation(signature.startBundle()))
+            .intercept(delegateOrNoop(signature.startBundle()))
```

Comment:
I'll ask again, only because it raised some questions -- should we always delegate let the BundleMethodDelegation deal with the implementation? For example, shouldn't startBundle always run the "mark aggregators as complete" (pre-delegation code) even if there is no `@StartBundle`?

Otherwise, I think I can create an aggregator in `@ProcessElement` in a `DoFn` that doesn't have a `@StartBundle`